src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -134,7 +134,7 @@ private void AddOrRemoveDropdown()
                 // for the local navbar implementation.
                 // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1163360
                 var service = document?.Project?.Solution?.Workspace?.Services?.GetRequiredService<IWorkspaceContextService>();
-                if (service != null && service.IsCloudEnvironmentClient() && document.Project.Language != "TypeScript")
+                if (service != null && service.IsCloudEnvironmentClient() && document!.Project!.Language != "TypeScript")
```

Comment:
If `service` isn't null then `document` and `document.Project` must not be null. @jcouv does this warrant a bug?